test/compliance_tool/test_compliance_check_aasx.py

```diff
@@ -130,3 +131,30 @@ def test_check_aasx_files_equivalence(self) -> None:
                       '[Identifier(IRI=https://acplt.org/Test_AssetAdministrationShell)] must be ==',
                       manager.format_step(4, verbose_level=1))
         self.assertEqual(Status.NOT_EXECUTED, manager.steps[5].status)
+
+    @unittest.skipIf(sys.version.startswith("3.6"), "The XML schema check fails for Python 3.6")
```

Comment:
As discussed in the meeting, the version number should be compared using the version tuple.